.readthedocs.yml

```diff
@@ -0,0 +1,26 @@
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required
+version: 2
+
+# Build documentation in the docs/ directory with Sphinx
+sphinx:
+  configuration: docs/source/conf.py
+
+# Build documentation with MkDocs
+#mkdocs:
+#  configuration: mkdocs.yml
+
+# Optionally build your docs in additional formats such as PDF
+formats:
+  - pdf
+
+# Optionally set the version of Python and requirements required to build your docs
+python:
+  version: 3.7
```

Comment:
Is this needed? Docs should build ok under any supported python version?

Reply:
They should, but [according to docs](https://docs.readthedocs.io/en/stable/config-file/v2.html#python-version) the default is just 3, and it's probably better to be explicit about it :)

Reply:
ah, thanks, best leave this in then!